phac-nml/irida-next
# Submit search on enter

## What does this PR do and why?
_Describe in detail what your merge request does and why._

1. Standardized Search Component
2. Search now works when the `enter` key is pressed, not debounced

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/user-attachments/assets/a8249184-bb63-40e4-ae0e-c7d3c2f59d33)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Ensure that search still works across the project (tested through our integration tests as well)

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
